src/decorator/columns/SoftDeleteDateColumn.ts

```diff
@@ -0,0 +1,23 @@
+import {ColumnOptions} from "../options/ColumnOptions";
+import {getMetadataArgsStorage} from "../../index";
+import {ColumnMetadataArgs} from "../../metadata-args/ColumnMetadataArgs";
+
+/**
+ * This column will store an soft delete date of the deleted object.
+ * This date is set when you delete the object, and set to null when you restore it
+ */
+export function SoftDeleteDateColumn(options?: ColumnOptions): Function {
```

Comment:
Do we have a better alternative names for this decorator? This looks correct but a bit long

Reply:
How about a simple `DeletedAt`?

Reply:
sounds better

Reply:
Should we then also change the `CreateDate` and `UpdateDate` decorators as well? I tried to conform to the existing decorators.

Reply:
ah you are right, its not consistent. For some reason I thought they are called `CreatedAt` and `UpdatedAt` haha. No, we can't rename them since we also need a `Column` postfix as used in all column decorators. @NoNameProvided any other suggestions for a more consistent decorator name?